scripts/build/libc/glibc.sh

```diff
@@ -102,7 +103,14 @@ do_libc_backend() {
     # If gcc is not configured for multilib, it still prints
     # a single line for the default settings
     multilibs=( $("${CT_TARGET}-gcc" -print-multi-lib 2>/dev/null) )
+    last_multi=
     for multilib in "${multilibs[@]}"; do
+        last_multi=$(( ${#multilibs[@]} - 1 ))
+        if [ "${multilib%%;*}" = "${multilibs[last_multi]%%;*}" ]; then
+            # This is the last multilib build or multilib is '.'
+            # (default target, not multilib)
+            multi_last=y
+        fi
```

Comment:
BTW...

Yes, it would need to be passed donw to the `do_libc_backend` function,
but rather than calling it `multi_last`, just call it `may_build_manuals`.